src/org.ts

```diff
@@ -211,6 +211,49 @@ export class Org extends AsyncCreatable<Org.Options> {
     }
   }
 
+  /**
+   * If this Org is a scratch org, calling this method will delete the scratch org from the DevHub and clean up any local files
+   *
+   * @param hubOrg - optional DevHub Org of the to-be-deleted scratch org
+   */
+  public async deleteScratchOrg(hubOrg?: Org): Promise<void> {
```

Comment:
is it up to the command to figure out if this is scratchOrg or not and then call the right method?

Reply:
that's what's currently happening. I was thinking about creating a general `delete` method that would figure it out and delegate